doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -2511,7 +2597,9 @@ tactic:
 The behavior of the defective have tactic makes it possible to
 generalize it in the following general construction:
 
-.. tacn:: have {* @i_item } {? @i_pattern } {? {| @s_item | {+ @ssr_binder } } } {? : @term } {? {| := @term | by @tactic } }
+.. tacn:: have {| suff | suffices } {? @ssripats } @ssrhavefwd
+          {| gen | generally } have {? @ssrclear } {? {| @ident | _ } , } {? @ssripats } @ssrwlogfwd {? by @ssrhintarg }
```

Comment:
Maybe I'm misreading but here the grammar should match
```
have ...
gen have ...
```
Is the line breaking, without a `|`, a disjunctive?

Reply:
It looks like this.  Hope you're looking at the HTML version of the doc--much easier to read!

![image](https://user-images.githubusercontent.com/1253341/97953932-e0bfef00-1d56-11eb-8f2a-ddda2c1a0f2a.png)

> Is the line breaking, without a `|`, a disjunctive?

I don't follow you.  Line 2601 is displayed across 2 lines.  The `tacn` directive requires that the entire syntax notation be on a single line.